bigbluebutton-html5/imports/startup/client/base.jsx

```diff
@@ -264,7 +264,6 @@ const BaseContainer = withTracker(() => {
     ejected: 1,
     ejectedReason: 1,
     color: 1,
-    mobile: 1,
```

Comment:
If you remove this line, I think you will be disabling the Mobile users altogether.. The mongo info about whether a user is mobile or not will not be available

Reply:
User info in the user list (the only place that uses `user.mobile` information) gets data from [userlist service](https://github.com/bigbluebutton/bigbluebutton/blob/develop/bigbluebutton-html5/imports/ui/components/user-list/service.js#L192), not from BaseContainer, so data will still be available